ner_v2/detectors/numeral/number_range/README.md

```diff
@@ -0,0 +1,246 @@
+## Number Range Detector 
+
+This detector module will help to detect minimum and maximum number values from text containing range values. For example - "200 to 300" (minimum=200 and maximum=300),  "more than 2000" (mininum=2000, maximum=Not define) 
+
+ We are currently providing number detection supports in 6 languages, which are
+
+- English
+- Hindi
+
+### Usage
+
+- **Python Shell**
+
+  ```python
+  >> from ner_v2.detector.number.number_range.number_range_detection import NumberRangeDetector
+  >> detector = NumberRangeDetector(entity_name='number_range', language='en')  # here language will be ISO 639-1 code
+  >> detector.detect_entity(text= 'I annual salary is 200k-500k rupees')
+  >> {'entity_value': [{'min_val': '200000', 'max_value': '500000', 'unit': 'rupees'}], 'original_text':['200k-500k rupees']}
+  ```
+
+- **Curl Command**
+
+  ```bash
+  # For a sample query with following parameters
+  # message="i want more than 12 mangoes"
+  # entity_name='number_range'
+  # structured_value=None
+  # fallback_value=None
+  # bot_message=None
+  # unit_type=None
+  # source_language='en'
+  # language_script='en'
+  
+  $ URL='localhost'
+  $ PORT=8081
+  
+  $ curl -i 'http://'$URL':'$PORT'/v2/number_range?message=do%20hajaar%20char%20sau&entity_name=number&structured_value=&fallback_value=&bot_message=&source_language=en&language_script=en&unit_type='
+  
+  # Curl output
+  $ {
+      "data": [
+          {
+              "detection": "message",
+              "original_text": "more than 12",
+              "entity_value": {
+                  "max_value": null,
+                  "min_value": "12",
+                  "unit": null
+              },
+              "language": "en"
+          }
+      ]
+  }
+  ```
+
+  
+
+### Steps to add new language for Number Range detection
+
+In order to add any new language you have to follow below steps:
+
+1. Create a directory with `ISO 639-1` code of that language inside `ner_v2/detectors/numeral/number_range/`.  
+
+2. Create a directory named `data` inside the language_code folder.
+
+3. Add a CSV file named `number_range_keywords.csv` inside data folder.  
+
+   Below is the folder structure of same after adding all the files for new language `xy`.
+
+   ```python
+   |__ner_v2
+         |___detectors
+             |___numeral
+                 |___number_range
+                     |___xy    # <- New language Added 
+                     |	  |___data
+                     |      |___number_range_keywords.csv
+                     |      
+                     |__number_detection.py 
+   ```
+
+
+
+
+####  GuideLines to create data files
+
+Below is the brief about how to create data file `number_range_keywords.csv` All the description of the file is explained using Hindi as a reference language. 
+
+1. **number_range_keywords.csv**:  This file contains the vocabs for range keywords, their position around number values and their range type.
```

Comment:
`vocabs for range keywords` -> `vocabulary used for mentioning ranges in the target language`